docs/core/get-started.md

```diff
@@ -20,8 +20,10 @@ Install .NET Core on [Windows](https://www.microsoft.com/net/core#windows).
 
 You can get started developing .NET Core apps by following these step-by-step tutorials.
 
-* [Building a C# Hello World Application with .NET Core in Visual Studio 2017](./tutorials/with-visual-studio.md) - Learn to to build, debug, and publish a simple .NET Core console application using Visual Studio 2017.
+* [Building a C# Hello World Application with .NET Core in Visual Studio 2017](./tutorials/with-visual-studio.md) - Learn to to build, debug, and publish a simple .NET Core console application using C# and Visual Studio 2017.
 * [Building a class library with C# and .NET Core in Visual Studio 2017](./tutorials/library-with-visual-studio.md) - Learn how to build a class library written in C# using Visual Studio 2017.
+* [Build a Visual Basic Hello World application with .NET Core in Visual Studio 2017](../../core/tutorials/vb-with-visual-studio.md) - Learn to to build, debug, and publish a simple .NET Core console application using Visual Basic and Visual Studio 2017. 
```

Comment:
build is giving you warnings for these new links. Probably should be ./tutorials